src/vs/workbench/browser/parts/titlebar/windowTitle.ts

```diff
@@ -85,20 +86,23 @@ export class WindowTitle extends Disposable {
 				this.titleUpdater.schedule();
 			}
 		}));
-		this.updateTitleIncludesFocusedView();
-	}
-
-	private updateTitleIncludesFocusedView(): void {
-		this.titleIncludesFocusedView = this.configurationService.getValue<string>(WindowSettingNames.title)?.match(/\${focusedView}/) !== null;
 	}
 
 	private onConfigurationChanged(event: IConfigurationChangeEvent): void {
-		if (event.affectsConfiguration(WindowSettingNames.title) || event.affectsConfiguration(WindowSettingNames.titleSeparator)) {
+		if (event.affectsConfiguration(WindowSettingNames.title)) {
```

Comment:
@meganrogge my understanding is that `updateTitleIncludesFocusedView` only needs to be called when `WindowSettingNames.title` is updated.